pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -923,27 +906,11 @@ func (pl *dynamicResources) Reserve(ctx context.Context, cs *framework.CycleStat
 	logger := klog.FromContext(ctx)
 	for index, claim := range state.claims {
 		if claim.Status.Allocation != nil {
```

Comment:
The if clause now wraps an empty block, prefer:

```go
if claim.Status.Allocation != nil {
  continue
}
// Must be delayed allocation.
numDelayedAllocationPending++
```

Reply:
Ack.

I didn't want to re-indent the entire else block, that would have obscured the small change that is needed there (the removal of `state.podSchedulingState.publish`). But as you have now seen the change, I can reformat as you suggested :grin:

Reply:
Actually, the else clause doesn't change at all here, the `state.podSchedulingState.publish` is further down. Re-indenting still makes sense.

Reply:
@Huang-Wei: [update](https://github.com/kubernetes/kubernetes/compare/98a0c9a199296019af8f5d096a403ab90b316e74..6cf42037512b266d9581a745461a131ddfe0da60) pushed, please take another look.